packages/contracts-bedrock/snapshots/storageLayout/L1ERC721Bridge.json

```diff
@@ -14,11 +14,25 @@
     "type": "bool"
   },
   {
-    "bytes": "1536",
-    "label": "__gap",
+    "bytes": "20",
+    "label": "messenger",
+    "offset": 2,
+    "slot": "0",
+    "type": "contract CrossDomainMessenger"
+  },
```

Comment:
This is being packed into the intializer slot, are we ok with that? It's not a breaking change but I recall discussing if we wanted to avoid this. If we never drop into assembly to modify slot 0 (and I don't think we do) this is safe, so to be clear I'm ok keeping it

Reply:
We will need to clear the initialized slot during the 2 step upgrade so we will either need to:
- re-set the messenger in the `initialize` call
- avoid zeroing it out by packing the call to `setStorage` correctly
- putting a spacer in the contract

I am fine with any of these options